api.ts

```diff
@@ -1406,6 +1409,12 @@ export interface SyncTableOptions<
    * sync tables that have a dynamic schema.
    */
   dynamicOptions?: DynamicOptions;
+  /**
+   * If the pack supports object updates, the maximum number of objects that will be sent to the pack
```

Comment:
Super nit, technically it's if the _table_ supports object updates. Other stuff in the pack isn't relevant.